pxtlib/tutorialValidator.ts

```diff
@@ -5,6 +5,7 @@ namespace pxt.tutorial {
         RuleTurnOn: boolean;
         RuleStatus: boolean;
```

Comment:
nit: usually fields are camelCased like isStrict; feel free to leave for later as it's not part of this pr anyways~